linux/docker_functions.bsh

```diff
@@ -481,8 +481,8 @@ function Just-docker-compose()
     local f
     #mimic docker-compose's default behavior
     compose_file_override=
-    for f in docker-compose.yml docker-compose.yaml \
-             docker-compose.override.yml docker-compose.override.yaml; do
+    for f in ./docker-compose.yml ./docker-compose.yaml \
```

Comment:
Could you add a comment in here to say that the `./` is just to fool [MINGW64's path translation](http://www.mingw.org/wiki/Posix_path_conversion)?